employee_seniority_level/views/hr_employee_seniority_level.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+  <record id="hr_employee_seniority_level" model="ir.ui.view">
+    <field name="name">hr.employee.form.seniority.level</field>
+    <field name="model">hr.employee</field>
+    <field name="inherit_id" ref="hr.view_employee_form"/>
+    <field name="arch" type="xml">
+      <field name="tz" position="after">
+        <field name="seniority_level_id"/>
+      </field>
+    </field>
+  </record>
+
+  <record id="product_template_seniority_level" model="ir.ui.view">
+    <field name="name">product.template.seniority.level</field>
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="product.product_template_form_view"/>
+    <field name="arch" type="xml">
+      <xpath expr="//field[@name='service_policy']" position="after">
+          <field name="seniority_level_id" attrs="{'required': [('service_policy', '=', 'delivered_timesheet')], 'invisible': ['|', ('type', '!=', 'service'), ('service_policy', '!=', 'delivered_timesheet')]}"/>
```

Comment:
Just wondering, shouldn't you add an onchange to empty seniority_level_id if service_policy is changed from delivered_timesheet to something else ?

Reply:
seems a good thing to do when hidden

Reply:
Onchange added